examples/configuration.nix

```diff
@@ -216,6 +216,12 @@
   # services.charge-lnd.policies = ''
   # '';
 
+  ### TEoS - The Eye of Satoshi
+  # Set this to enable teos, a Lightning watchtower compliant with BOLT13,
+  # written in Rust.
+  #
+  # services.teos.enable = true;
+
```

Comment:
This could be an example configuration in case you don't want to touch the modules/onion-services.nix but still want use tor to expose the teosd endpoint.
```
    # teos
    nix-bitcoin.onionServices.teos = {
      enable = true;
      externalPort = 2121;
    };
    services.teos.enable = true;
```